### PR TITLE
Extend current self_update_id doc.

### PR DESCRIPTION
**Pull request by teclator**

### CODE REVIEW

File: doc/SELF_UPDATE.md [hunk 1]
```diff
@@ -111,9 +111,15 @@ The URL of the update repository is evaluated in this order:
         required just like in manual installation.
    4. Default SUSE Customer Center API (`https://scc.suse.com/`).
 
-   Registration server is then asked for URL providing product on media.
-   For multi-product media it is possible to specify ID used instead of product
-   name in control.xml as globals->self_update_id.
+   The registration server is then asked for the list of update repositories.
+
+   For being able to determine the list, the registration server expect
```
Comment (imobachgs) [inline doc/SELF_UPDATE.md]: What about "In order to determine" ?

Comment (imobachgs) [reply doc/SELF_UPDATE.md]: I would dot not use a blank line and, instead, I would say something like: *In order to determine such a list, the product `name`, `version` and `arch` are used.*

File: doc/SELF_UPDATE.md [hunk 2]
```diff
@@ -111,9 +111,15 @@ The URL of the update repository is evaluated in this order:
...
+   the product `name`, `version` and `arch` of the installer being updated.
+   **Ex:** `product: SLED`, `version: 15`, `arch: x86_64`
```
Comment (jreidinger) [inline doc/SELF_UPDATE.md]: I prefer to not use accronyms, it is just confusing.

File: doc/SELF_UPDATE.md [hunk 3]
```diff
@@ -111,9 +111,15 @@ The URL of the update repository is evaluated in this order:
...
+
+   In case of multi-product media, as the installer is the same for all the
```
Comment (imobachgs) [inline doc/SELF_UPDATE.md]: what about *a multi-product installation medium*?

File: doc/SELF_UPDATE.md [hunk 4]
```diff
@@ -111,9 +111,15 @@ The URL of the update repository is evaluated in this order:
...
+   products on the media, the product `name` can be hard-coded in the control
```
Comment (imobachgs) [inline doc/SELF_UPDATE.md]: *for all the included products* (and you can get rid of *media*).

File: doc/SELF_UPDATE.md [hunk 5]
```diff
@@ -111,9 +111,15 @@ The URL of the update repository is evaluated in this order:
...
+   file (control.xml) as `globals -> self_update_id`.
```
Comment (imobachgs) [inline doc/SELF_UPDATE.md]: I would keep consistency and use "xpath" style: `/globals/self_update_id` (as we do for `/general/self-update-url`).

**Pull request merged**